app/scripts/HiGlassComponent.js

```diff
@@ -359,6 +363,13 @@ class HiGlassComponent extends React.Component {
     }
   }
 
+  get sizeMode() {
+    // eslint-disable-next-line no-nested-ternary
+    return typeof this.props.options.sizeMode === 'undefined'
```

Comment:
this would probably be much clearer as an if statement.

Reply:
Mhhh the if statement would look the same and it would lead to code duplication, which is the reason why I turned it into a function.